app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -475,7 +481,35 @@ private fun MessageContent(
                     messageBody = messageContent.messageBody,
                     isAvailable = !message.isPending && message.isAvailable,
                     onLongClick = onLongClick,
-                    onOpenProfile = onOpenProfile
+                    onOpenProfile = onOpenProfile,
+                    buttonList = null,
+                    onButtonClick = null,
+                    pendingButton = null,
+                    messageId = message.header.messageId
```

Comment:
In my personal opinion it would be create to create a seperate MessageBody component that would be a Composite. I read better when u have a seperate component and see exactly what it takes in as component instead of diving into the function and looking for the if and null checks

Reply:
My idea behind this is to handle it as a normal text message because it is just a text message with buttons.